.github/workflows/test-code.yml

```diff
@@ -10,13 +10,8 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php-version: [ 7.2, 7.3, 7.4 ]
-                shopware-version: [ \~5.6, \~5.7 ]
-                exclude:
-                    - php-version: 7.2
-                      shopware-version: \~5.7
-                    - php-version: 7.3
-                      shopware-version: \~5.7
+                php-version: [ 7.4 ]
```

Comment:
evtl. dann php 8 hinzufügen wenn du das unten in der json als supported angibts

Reply:
Der aktuelle Coding Standard unterstützt kein php8.0 https://github.com/netlogix/nlxBackToTopButton/runs/8091096904?check_suite_focus=true